client/js/constants.js

```diff
@@ -1,5 +1,24 @@
 "use strict";
 
+const colorCodeMap = [
+	[0, "White"],
```

Comment:
Ok did that, it's much cleaner like this and puts the mapping of color codes/names as a source of truth in the constants.
I could also reconstruct the codes from index + leading zero + `toString` but it feels hacky. Better to keep the codes here, and also makes it possible to support future stuff like `"99"`.